pkg/provider/okta/okta.go

```diff
@@ -371,7 +371,15 @@ func verifyMfa(oc *Client, oktaOrgHost string, resp string) (string, error) {
 			"Passcode",
 		}
 
-		duoMfaOption := prompter.Choose("Select a DUO MFA Option", duoMfaOptions)
+		duoMfaOption := 0
+
+		if loginDetails.DuoMFAOption == "Duo Push" {
```

Comment:
I'm not sure this is safe, I don't think there's any certainty of the order - you may need to iterate over each and compare

Reply:
I assumed it was OK, since the code previously assumed an order (and since the array is declared just above)

I could iterate to be make it safer however.